src/components/LoginUser.js

```diff
@@ -50,7 +50,7 @@ function LoginUser() {
                 </form>
                 {loginMsg && <p>Successfully logged in!</p>}
```

Comment:
one more thing: don't use an exclamation point

Reply:
Story required exclamation point

Reply:
oh right, I forgot about that